compiler/il/OMROpcodes.enum

```diff
@@ -6106,7 +6106,7 @@ OPCODE_MACRO(\
    /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE, \
    /* .properties3          = */ 0, \
    /* .properties4          = */ 0, \
-   /* .dataType             = */ TR::VectorInt32, \
+   /* .dataType             = */ OMR::DataType::Vector128Int32, \
```

Comment:
Are these vector opcodes that may be re-designed in the future but are being left as per the original behavior in this PR ? Just wondering about the specific datatype in use for these (as opposed to the different approach for say a `vadd` opcode).

Reply:
OMR::DataType::Vector128Int32 is a temporary static variable which will not be used.

Reply:
Thanks, I imagined this must be the case but wanted to double check with you. I have approved the PR since that was the last question I had from the review.